### ISSUE TITLE
Provide release notes

### ISSUE THREAD

**Issue by JobaDiniz** [lgalfaso/angular-dynamic-locale#54]
Please, provide release notes for each release so we know what is fixed, which new feature is added, and most importantly, whether it has any breaking changes.

**Comment by starf** [comment]
This, because with 0.1.27 i have no idea what has changed/improved.
Thanks

**Comment by JobaDiniz** [comment]
I have 0.1.17. Almost every release does not have notes... Dont know whether has breaking changes

**Comment by lgalfaso** [comment]
I have to agree on this, and also need to automate it. BTW, there are no breaking changes since the first release, only small additions done every now and them.

Will keep this open to track automating the change log and release notes
